npm-shrinkwrap.json

```diff
@@ -6,7 +6,7 @@
       "version": "1.0.3"
     },
     "abbrev": {
-      "version": "1.0.7"
+      "version": "1.0.9"
```

Comment:
Why is shrinkwrap changing versions on modules we haven't touched?  I thought the point of it was to stop that happening?